core_backend/app/question_answer/routers.py

```diff
@@ -202,7 +205,7 @@ async def feedback(
             },
         )
```

Comment:
Remove space after colon above to say "Secret key does not match query id:"

Reply:
My friend...

Reply:
Ok. Fixed